### ISSUE TITLE
file share bug color

### ISSUE THREAD

**Issue by xi7** [gsantner/markor#2134]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**. Not a [question or feature request](https://github.com/gsantner/markor/issues/new/choose).
- [X] The topic is **not** already reported at [Issues](https://github.com/gsantner/markor/issues?q=#js-issues-search). _(I've searched it)_.
- [X] Markor **is** up to date. See [Releases](https://github.com/gsantner/markor/tags) for the latest version. Updates are available from [F-Droid](https://f-droid.org/en/packages/net.gsantner.markor/) and GitHub.
- [X] The bug is still present in the latest development version (git master). (Please [download](https://nightly.link/gsantner/markor/workflows/build-android-project/master) and try the test version of Markor, named **Marder**. Don't worry; Markor and Marder appear as completely separate applications. You can install both side-by-side, and Markor settings are not touched. In case the issue is resolved there, you don't need to create a bug report. The change will be part of the next Markor update.)

### Description

1) after share to any file the color background always change to white.

2) share to image need be different of screenshot. replace save image to full copy table and not just like screenshot or will cut what not see.

### Steps to reproduce

1. click 3 dots
2. share
3. select option screenshot. (test)

### Information

Android version: 11
Device: asus
App Version: latest

### Source

GitHub Releases

### Format / File type

Markdown

### Additional info / Log

```shell
-
```

**Comment by gsantner** [comment]
It is intentional. Export is also in the meaning of a printable version.

**Comment by xi7** [comment]
but the view is black background, and print show white different, not possible u add save to image same 
what show background ?

full image and like screenshot.

**Comment by xi7** [comment]
if people want print device pdf white is ok. 

but all option share is not good idea convert to white.

**Comment by gsantner** [comment]
Makes sense to reduce the automatic-light export to specific export methods

**Comment by xi7** [comment]
what if u add option ViewMode to png full size ?
